workflow-bot-app/src/main/java/com/symphony/bdk/workflow/security/AuthorizationAspect.java

```diff
@@ -16,7 +16,6 @@
 public class AuthorizationAspect {
 
   private static final String UNAUTHORIZED_EXCEPTION_INVALID_TOKEN_MESSAGE = "Request token is not valid";
```

Comment:
We can then make the error message more generic, like "Request is not authorised."